libenv/sysinfo.c

```diff
@@ -456,7 +456,7 @@ static void GetNameInfo3(EvalContext *ctx)
     snprintf(real_version, _SYS_NMLN, "%.80s.%.80s", VSYSNAME.version, VSYSNAME.release);
     strlcpy(VSYSNAME.release, real_version, _SYS_NMLN);
 #endif
-#ifdef __ANDROID__
+#if defined(__ANDROID__) && !defined(__TERMUX__)
```

Comment:
Shouldn't there be `strcpy(VSYSNAME.sysname, "termux");` for *termux* then?

Reply:
`VSYSNAME.sysname` is used for the `os` class and that is still Android.
I consider that termux is more of a flavor of Android and not the OS itself.
By putting `!defined(__TERMUX__)` I get "linux" which is what I want so I don't have to re-write lots of strings in libpromises/systype.c (all the commands and command options and such). All that "linux" stuff works well with the default packages installed with termux.